Packs/CortexXDR/Integrations/CortexXDRIR/README.md

```diff
@@ -690,6 +690,7 @@ Builtin Roles with this permission includes: "Privileged Responder", "Viewer" an
 | endpoint_id_list | A comma-separated list of endpoint IDs. | Optional | 
 | dist_name | A comma-separated list of distribution package names or installation package names.<br/>Example: dist_name1,dist_name2. | Optional | 
 | ip_list | A comma-separated list of IP addresses.<br/>Example: 8.8.8.8,1.1.1.1. | Optional | 
```

Comment:
The readme is not correlated with the .yml description.

Reply:
fixed in [GR100 and README](https://github.com/demisto/content/pull/30308/commits/8d540f0dd9399b9aaa9c092140d2bff00ff99e18)